### ISSUE TITLE
improve Text and UI 

### ISSUE THREAD

**Issue by nik132-eng** [coding-geek21/classDeck#100]
## Prerequisites
- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

## Define You

- [X] GSSoC22 Participant
- [X] Contributor

## Is your feature request related to a problem? Please describe.
there is better text available in the market so I can change it all.

## Approach to be followed (optional):
change UI and text

**Comment by nik132-eng** [comment]
please assign me this issue so i can work on this

**Comment by navyasharma0203** [comment]
Can I work on this issue??

**Comment by coding-geek21** [comment]
@nik132-eng Any updates?

**Comment by nik132-eng** [comment]
![image](https://user-images.githubusercontent.com/77881638/169221882-f6a06465-6b8e-472e-98f2-aba13b899ad4.png)

Sorry 😓

**Comment by nik132-eng** [comment]
in my opinion, you have to give some guidance and time to beginners.

no worries thank you for your time @coding-geek21

**Comment by coding-geek21** [comment]
> in my opinion, you have to give some guidance and time to beginners.
> 
> no worries thank you for your time @coding-geek21

Yeah but this is not beginner level project and this is completely different from java.. So you have to first learn django if you want to contribute to it and if you are familiar with html and css. then you can contribute on that part @nik132-eng
